packages/synapse-react-client/src/components/CreateTableViewWizard/TableNameForm.tsx

```diff
@@ -16,7 +18,7 @@ type TableNameFormProps = {
 export default function TableNameForm(props: TableNameFormProps) {
   const { name, setName, description, setDescription } = props
   const { isInExperimentalMode } = useSynapseContext()
-
+  const { data: featureFlags } = useGetFeatureFlags()
```

Comment:
Could use the suggested `useGetFeatureFlag(FeatureFlagEnum.DESCRIPTION_FIELD)` (see comment on `useGetFeatureFlags.ts`)